jenkins/shared-library/vars/defaultCiPipeline.groovy

```diff
@@ -212,10 +212,10 @@ void call(Closure body) {
 							allOf {
 								expression {
 									// The branch indexing build TEST_MODE = null
-									return env.TEST_MODE == null || 'code-coverage' == env.TEST_MODE
+									return null == params.TEST_MODE || 'code-coverage' == params.TEST_MODE
```

Comment:
why this change from env to params?

Reply:
This was more of a cleanup of code.  Should probably be in a different PR 😅 
Parameters selected by the users can be accessed either by ``env`` or ``params``.  I am switching these to use just ``params`` for readability.